.github/workflows/release.yml

```diff
@@ -62,8 +58,6 @@ jobs:
         uses: micronaut-projects/github-pages-deploy-action@grails
         env:
```

Comment:
The micronaut-projects/github-pages-deploy-action does not really use gradle so the whole env block can be removed.

Reply:
@matrei Cool, thanks! What about `TARGET_REPOSITORY`, `GH_TOKEN`, and `BRANCH`? Aren't these used?

Reply:
Yes, sorry, that was wrong to say the whole env block could be removed. Only the GRADLE_ENTERPRISE_* vars can be removed.

Reply:
@matrei No worries 🙂 It's good to know that we can drop all those GRADLE_ENTERPRISE_* env vars. Thank you!